core/src/layers/correctness_check.rs

```diff
@@ -0,0 +1,602 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use std::fmt::{Debug, Formatter};
+use std::sync::Arc;
+
+use crate::raw::{
+    Access, AccessorInfo, Layer, LayeredAccess, OpBatch, OpCopy, OpCreateDir, OpDelete, OpList,
+    OpPresign, OpRead, OpRename, OpStat, OpWrite, Operation, RpBatch, RpCopy, RpCreateDir,
+    RpDelete, RpList, RpPresign, RpRead, RpRename, RpStat, RpWrite,
+};
+use crate::{Error, ErrorKind};
```

Comment:
The same, use `crate::*` instead.

Reply:
why do we favor using `crate::*` here ?

Reply:
> why do we favor using `crate::*` here ?

Simple and easy to maintain, OpenDAL's exported APIs are well-designed. We never export anything arbitrarily, so we can rely on them.